Bottom: 160c1e0303cd87de74bd5816bfdc081973cc7d45
Top:    c3de225da83869df8c17f28821b440c8fe6eb968
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-14 00:57:43 +0000

Refresh of mess-of-attempts-to-debug-and

---

diff --git a/src/polygon1.c b/src/polygon1.c
index ce673da..9af013a 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -760,12 +760,6 @@ cvc_list_dump (CVCList *list)
   VNODE *node = list->parent;
   CVCList *iter;
 
-  /* Only print for the vertex we're interested in at the moment */
-  if (!(
-      (node->point[0] == 69088001 || node->point[0] == 69088000 || node->point[0] == 69087998) &&
-      node->point[1] == 29336999))
-    return;
-
   pcb_fprintf (stderr, "Dumping CVC list at (%$mn, %$mn)\n", node->point[0], node->point[1]);
 
   iter = list;
